README.md

```diff
@@ -51,10 +42,11 @@ See [the list of releases][6] to find out about feature changes.
 [11]: https://kubernetes.io/docs/tasks/tools/install-kubectl/#tabset-1
 [12]: https://github.com/kubernetes/kubernetes/blob/master/cluster/addons/dns/README.md
 [14]: https://github.com/kubernetes/kubernetes
+[17]: https://velero.io/docs/master/restic.md
 [24]: https://groups.google.com/forum/#!forum/projectvelero
 [25]: https://kubernetes.slack.com/messages/velero
 [28]: https://velero.io/docs/install-overview
 [29]: https://velero.io/docs/
 [30]: https://velero.io/docs/troubleshooting
 [99]: https://velero.io/docs/support-matrix
-[100]: /site/docs/master/img/velero.png
+[100]: https://velero.io/docs/master/img/velero.png
```

Comment:
Doesn't look like this is used anywhere, no need to add it

Reply:
It's on line 1.

Reply:
Ah. I don't think it needed to change, but I guess it's fine to hit the website.